### ISSUE TITLE
Tab Group replacement: Sync Tab Groups

### ISSUE THREAD

**Issue by erotavlasme** [Quicksaver/Tab-Groups#565]
Hi,
I just want to advise an add on [Sync Tab Groups](https://morikko.github.io/synctabgroups/) that has already the majority (95%) of features of Tab Group and it supports firefox quantum i.e. 57+.
In my opinion the problem has been solved.

**Comment by dgutov** [comment]
@erotavlas85 Does it use the tab hiding API? Or it reloads the tabs on each group change?

**Comment by erotavlasme** [comment]
@dgutov it reloads the tabs on each group change. If you want tabs hiding, you can use [Conex](https://addons.mozilla.org/en-US/firefox/addon/conex/), but it requires firefox 60+ and manually enabling extensions.webextensions.tabhide.enabled. In my opinion, at the moment Sync Tab Group is the best solution.

**Comment by dgutov** [comment]
Conex is tied to containers. I've been using https://addons.mozilla.org/en-US/firefox/addon/experimental-simple-tab-groups/, with moderate success.

**Comment by erotavlasme** [comment]
@dgutov thank you for the advice. However, I just read that Sync Tab Group has [experimental support for tab hiding](https://github.com/Morikko/sync-tab-groups/wiki/Hiding-support) and in the future also [container support]( https://github.com/Morikko/sync-tab-groups/issues/52).

**Comment by ExploitSage** [comment]
I have been using [Panorama View](https://addons.mozilla.org/en-US/firefox/addon/panorama-view/), and while it still has the occasional bug (mostly with moving tabs between windows recently), it has been very functional, has most of important features Tab Groups had, and isn't tied to containers. You do still have to manually set **extensions.webextensions.tabhide.enabled** to **true** to enable tab hiding.

**Comment by arganoid** [comment]
I'm currently on Firefox 52 ESR. According to the release calendar the ESR will update to version 60.2 on 5th September. What is the state of play regarding tab groups replacements – how usable/stable are they, and is there one that stands out as being noticeably better than the others? Finally, what happens to my existing tabs and groups when the upgrade occurs? Of course, there won't be any groups but do the tabs get kept or lost? Can the groups be imported into one of the new extensions?

I've just had a look at Panorama View, but it hasn't been updated since April.

**Comment by ExploitSage** [comment]
At this point, the most usable Tab Group replacement seems to have become [Simple Tab Groups](https://addons.mozilla.org/en-US/firefox/addon/simple-tab-groups/). It is still being updated regularly, it is quick, easy to use, and not tied to containers. It handles multiple firefox windows VERY well. It uses a pop-up window for the quick stuff, but does have a full-tab view you can access as well. It also handes "lost" tabs very well, displaying them in the address-bar menu with options to close them or move them into groups. I have been using simple tab groups at work for over a month and just installed it at home about 2 weeks ago.

**Comment by erotavlasme** [comment]
I agree with @gurustave at the moment Simple Tab Groups has overcome Sync Tab Group.
Another interesting plug in for tab management and session management is [Session Sync](https://addons.mozilla.org/en-US/firefox/addon/session-sync/).

**Comment by BenjamenMeyer** [comment]
Trying out Simple Tab Groups - upgraded one of my installs of FF from FF v56 to v61; unfortunately, import of old tab groups did not carry over very well...so now I gotta re-org 500+ tabs :(

Otherwise functionality seems fine and comparable.

**Comment by arganoid** [comment]
In preparation for the upgrade, I have been using Bookmark All Tabs to mirror my groups into bookmark folders. On 3 Sep 2018 07:51, Benjamen Meyer <notifications@github.com> wrote:Trying out Simple Tab Groups - upgraded one of my installs of FF from FF v56 to v61; unfortunately, import of old tab groups did not carry over very well...so now I gotta re-org 500+ tabs :(

**Comment by mausalimi** [comment]
sadly panorama view seems no longer maintained as well. :(

**Comment by janderson099** [comment]
> 
> 
> sadly panorama view seems no longer maintained as well. :(

Indeed. Not working. :(

**Comment by ExploitSage** [comment]
Unfortunate to hear about Panorama View. Thankfully, I've still been very happy with Simple Tab Groups.

**Comment by erotavlasme** [comment]
At the moment, I think that the best solution is still [Session Sync](https://addons.mozilla.org/en-US/firefox/addon/session-sync/). The most important [missing feature](https://github.com/ReDEnergy/SessionSync/issues/40) are containers.

**Comment by BenjamenMeyer** [comment]
Update: I've come to love Simple Table Groups, especially the multi-window capability so I can have different tab groups in different windows, and if I close a window not worry about losing them (like was the case previously with TabGroups/Panorama).

**Comment by luigifab** [comment]
I dream or Chrome is integrating natively an equivalent of Tab Groups 1? [Read more on chromestory.com](https://www.chromestory.com/2019/01/chrome-tab-groups-will-sync-across-devices/).
